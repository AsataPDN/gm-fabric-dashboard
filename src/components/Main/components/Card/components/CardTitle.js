import styled from "styled-components";

import { FONT_SIZE_SM, FONT_STACK_DATA } from "style/styleVariables";

const CardTitle = styled.div`
  font-size: ${FONT_SIZE_SM};
  font-family: ${FONT_STACK_DATA};
  font-weight: 500;
  min-height: 1em;
`;

export default CardTitle;
