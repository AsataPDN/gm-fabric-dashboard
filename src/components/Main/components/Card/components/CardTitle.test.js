import React from "react";
import { shallow } from "enzyme";
import CardTitle from "./CardTitle";

describe("CardTitle", () => {
  it("matches snapshot", () => {
    const cardTitleWrapper = shallow(<CardTitle />);
    expect(cardTitleWrapper).toMatchSnapshot();
  });
});
