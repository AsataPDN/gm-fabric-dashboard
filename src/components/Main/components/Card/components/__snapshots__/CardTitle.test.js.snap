// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle matches snapshot 1`] = `
.c0 {
  font-size: 11px;
  font-family: Rubik,-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
  font-weight: 500;
  min-height: 1em;
}

<div
  className="c0"
/>
`;
